packages/lesswrong/server/resolvers/userResolvers.ts

```diff
@@ -65,6 +79,7 @@ addGraphQLResolvers({
           username,
           displayName: username,
           slug: await Utils.getUnusedSlugByCollectionName("Users", slugify(username)),
+          email,
```

Comment:
Just want to check my understanding: so when `email` is updated here, `emails` also gets updated because of the function `usersEditCheckEmail` in `userCallbacks.ts`, right?

Reply:
Correct.